### DISCUSSION TITLE
Terraforming - Help<title>

### DISCUSSION THREAD

**Discussion by SeraSte** [community/community#39683] [Projects and Issues]
### Select Topic Area

General

### Body

Buon pomeriggio
sono Stefano, volevo fare i complimenti per la realizzazione del sito open source di Terraforming Mars:
https://terraforming-mars.herokuapp.com/

Vista la così bella e tanto utile situazione volevo offrirvi il mio aiuto nel segnalarvi, nei limiti del possibile, gli errori o le non traduzioni di alcune delle carte che vengono giocate.
Sarebbe bello, non so se lo avete già fatto, realizzare altre giochi da tavolo open source.
In attesa di un vostro contatto mail, vi ringrazio.
Distinti saluti, Stefano.

**Comment by ossanna16** [comment]
Hi there @SeraSte and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
